tlslite/keyexchange.py

```diff
@@ -274,6 +275,9 @@ def processClientKeyExchange(self, clientKeyExchange):
             raise TLSLocalAlert(AlertDescription.illegal_parameter,
                                 "Invalid dh_Yc value")
 
+        if dh_Yc % self.dh_p == 0:
+            raise TLSIllegalParameterException("Suspicious dh_Yc value")
```

Comment:
So delete that condition, yes? And I didnt find any test for that `if not 2 <= dh_Yc <= self.dh_p - 1:` in `unit_tests/test_tlslite_keyexchange.py`, so I am not sure, if theres any test for it. Should I add that test?